easybuild/easyblocks/w/wrf.py

```diff
@@ -112,6 +113,11 @@ def configure_step(self):
         else:
```

Comment:
@boegel I don't really understand this part of the code.  It seems to me the `if not (hdf5 or parallel_hdf5)` can never be triggered, since we're already inside the `if hdf5` which means that `(hdf5 or parallel_hdf5)` will always be `True` which means that the `if not` will never be triggered. I'm aware that this piece of the code is years old, but do you happen to remember why this was written that way?

Reply:
Yeah, that line is wrong, it should be (note the change from error to info)
```
if not parallel_hdf5:
  self.log.info("Parallel HDF5 module not loaded, assuming that's OK...")
else:
  ...
```
This is assuming that the PHDF5 variable is still relevant for WRF.

Reply:
there's a library `external/io_phdf5` which is still part of WRF.  So I'm assuming it's still relevant.

I'll fix this one later whem I'm at work and update this PR.